changelogs/unreleased/0001-dsmithuchida

```diff
@@ -0,0 +1,3 @@
+Added astrolabe.S3Config to fully specify the Astrolabe S3 server.  This allows
```

Comment:
Can we use PR number as the prefix of the changelog file?

i.e. `0024-dsmithuchida`

Reply:
Sure, that's easy enough